### PR TITLE
Upgrade from Ubuntu 20.04 to 22.04 in Head, Uyuni and 4.3

### PR DESCRIPTION
**Pull request by srbarrios**
Part of https://github.com/SUSE/spacewalk/issues/18408

### CODE REVIEW

File: terracumber_config/tf_files/SUSEManager-4.3-build-validation.tf [hunk 1]
```diff
@@ -248,7 +248,7 @@ module "base_debian" {
   name_prefix = "suma-bv-43-"
   use_avahi   = false
   domain      = "mgr.prv.suse.net"
-  images      = [ "ubuntu1804o", "ubuntu2004o", "ubuntu2204o", "debian10o", "debian11o" ]
+  images      = [ "ubuntu1804o", "ubuntu2204o", "ubuntu2204o", "debian10o", "debian11o" ]
```
Comment (vandabarata) [inline terracumber_config/tf_files/SUSEManager-4.3-build-validation.tf:251]: you have it duplicated here
```suggestion
  images      = [ "ubuntu1804o", "ubuntu2204o", "debian10o", "debian11o" ]
```

File: terracumber_config/tf_files/SUSEManager-Test-Orion-NUE.tf [hunk 2]
```diff
@@ -201,13 +201,13 @@ module "cucumber_testsuite" {
...
-      name = "min-ubuntu2004"
-      image = "ubuntu2004o"
+      name = "min-ubuntu2204"
+      image = "ubuntu2204o"
       provider_settings = {
         mac = "aa:b2:93:01:00:7b"
       }
       additional_repos = {
-        Salt_repo = "https://download.opensuse.org/repositories/systemsmanagement:/saltstack:/bundle:/testing:/Ubuntu2004/Ubuntu_20.04/"
+        Salt_repo = "https://download.opensuse.org/repositories/systemsmanagement:/saltstack:/bundle:/testing:/ubuntu2204/Ubuntu_20.04/"
```
Comment (vandabarata) [inline terracumber_config/tf_files/SUSEManager-Test-Orion-NUE.tf:210]:
```suggestion
        Salt_repo = "https://download.opensuse.org/repositories/systemsmanagement:/saltstack:/bundle:/testing:/ubuntu2204/Ubuntu_22.04/"
```
could that be the correct link instead?

File: terracumber_config/tf_files/SUSEManager-4.3-build-validation.tf (hunk 1 above)
Comment (srbarrios) [reply terracumber_config/tf_files/SUSEManager-4.3-build-validation.tf:251]: Oups! Right! Too fast search-replace :)

File: terracumber_config/tf_files/SUSEManager-Test-Orion-NUE.tf (hunk 2 above)
Comment (vandabarata) [reply terracumber_config/tf_files/SUSEManager-Test-Orion-NUE.tf:210]: actually, it seems like it isn't. but the correct one is different anyway
```suggestion
        Salt_repo = "https://download.opensuse.org/repositories/systemsmanagement:/saltstack:/bundle:/testing:/Ubuntu2204/Ubuntu_22.04/"
```

Comment (srbarrios) [reply terracumber_config/tf_files/SUSEManager-Test-Orion-NUE.tf]: Right!

**Review by vandabarata — CHANGES_REQUESTED** [review]

File: terracumber_config/tf_files/SUSEManager-Test-Orion-NUE.tf [hunk 3]
```diff
@@ -201,13 +201,13 @@ module "cucumber_testsuite" {
...
-      name = "min-ubuntu2004"
-      image = "ubuntu2004o"
+      name = "min-ubuntu2204"
+      image = "ubuntu2204o"
       provider_settings = {
         mac = "aa:b2:93:01:00:7b"
       }
       additional_repos = {
-        Salt_repo = "https://download.opensuse.org/repositories/systemsmanagement:/saltstack:/bundle:/testing:/Ubuntu2004/Ubuntu_20.04/"
+        Salt_repo = "https://download.opensuse.org/repositories/systemsmanagement:/saltstack:/bundle:/testing:/ubuntu2204/Ubuntu_22.04/"
```
[the remark this answers is not in the document]
Comment (srbarrios) [reply terracumber_config/tf_files/SUSEManager-Test-Orion-NUE.tf]: omg, thank you

**Review by vandabarata — APPROVED** [review]

**Pull request merged**